knitr/planetary_motion/planetary_motion.rmd

```diff
@@ -0,0 +1,735 @@
+Practitioners need to take bespoke steps to fit, evaluate, and improve their models.
+At the same time, the more modeling we do, the better prepared we usually are for the next project we undertake.
+It's not uncommon to apply hard-learned lessons from a past project to a new problem.
+Articles on _modeling workflows_ aim to formalize this experience, building and deconstructing heuristics, developing methods, and motivating new theory [e.g. @Blei:2014; @Gabry:2019; @Betancourt:2020; @Gelman:2020].
+We believe this effort must be complemented by case studies, such as this one, which illustrate in great details the concepts that arise in workflows, provide usable code, and also raise new questions.
+The specific concepts we encounter in this article include why our inference fails way more slowly than it succeeds, why the behavior of an ODE changes wildely across the parameter space, and what to do when chains drift into "unreasonable" modes of the posterior distribution.
+
+Given the recorded position of a planet over time, we want to estimate the physical properties of a star-planet system.
+This includes position, momentum, and gravitational interaction.
+We fit the model with Stan [@Carpenter:2017], using Hamiltonian Monte Carlo (HMC), a gradient-based Markov chain Monte Carlo (MCMC) algorithm;
+for a thorough introduction to HMC, we recommend the article by @Betancourt:2018.
+Initially, we meant the planetary motion problem to be a simple textbook example for ODE-based models; but it turns out many interesting challenges arise when we do a Bayesian analysis on this model.
+We discuss how to diagnose and fix these issues.
+The techniques we deploy involve a combination of mathematical considerations that are specific to the system at hand and statistical principles we can generalize.
+This case study examines the model in the controlled setting of simulated data.
+
+We begin by constructing our model, using elementary notions of classical mechanics.
+After failing to fit this complete model, we develop a simpler model and build our way back to the original model.
+This step-by-step expansion allows us to isolate issues that make Bayesian inference challenging.
+Classic tools we use involve: running multiple chains, convergence diagnostics, and simulating data from the fitted model.
+The latter can be done in several ways and an agile use of plots proves extremely helpful.
+Running simulations for multiple parameter values allows us to understand how the parameters interact with the likelihood, and identify local modes in the posterior.
+In our presentation, we try to distinguish generalizable methods, problem-specific steps, and shortcoming in existing defaults.
+
+
+## R setup {-#setup}
+
+```{r message = FALSE}
+# Adjust to your setting
+.libPaths("~/Rlib/")
```

Comment:
Assume defaults here

Reply:
This should run with default imports and whatnot. Just like if someone comes into the folder and presses "knit" it should knit. But I'll figure it out once the RDS's are there.